### DISCUSSION TITLE
发帖需要梯子，可以阻拦一些无脑人群，我很认可

### DISCUSSION THREAD

**Discussion by koneah** [share121/inter-knot#306] [其他]
太棒了

**Comment by xxtg666** [comment]
是这样的
（其实只是链接不稳定 不是一定要）

**Comment by SteveXXXXiao** [comment]
其实说到底还是国内网络环境的问题，包括现在的一些节奏问题，能做到这样子有一个基础的筛选我觉得确实很不错，期待作者大大可以继续保持了

**Comment by dingzhen-vape** [comment]
有道理

**Comment by capacitor1** [comment]
想必各位有梯子的绳匠一定有用梯子的真实目的吧doge
就比如[InterKnotForm@292 - 涩涩，我要涩涩](https://github.com/share121/inter-knot/discussions/292)
doge

**Comment by hooymie** [comment]
梯子其实门口不高吧，反正e站啊，粉色漫画软件都不敢看评论区的，全是吵架

**Reply by koneah** [reply]
梯子门槛不高，但是那些什么女拳，故意引战的会少很多

**Comment by kianaS0** [comment]
其实梯子也不一定能拦感觉，Steam++现在挺好下的

**Comment by Zhishirur** [comment]
不是也可以用加速器吗？虽然我也用的魔法。

**Reply by koneah** [reply]
加速器和梯子是一样的，另一个称呼而已

**Comment by moonhalo7** [comment]
我没用梯子，就有点慢，其他没多大影响

**Comment by cpu644** [comment]
你说的对，但是原礻
